packages/launchpad/src/launchpad-container/style.scss

```diff
@@ -0,0 +1,54 @@
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/breakpoints";
+
+.launchpad-container {
+	height: 100%;
+	display: flex;
+	flex-direction: column;
+}
+
+// Launchpad Content.
+.launchpad-container__content {
+	display: flex;
+	flex-direction: column;
+
+	@include break-large {
+		height: 100%;
+		min-height: calc(100vh - 60px); // Discount margin-top from `.step-route`.
+		flex-direction: row;
+	}
+}
+
+// Launchpad Sidebar.
+.launchpad-container__sidebar {
+	display: flex;
+	flex-direction: column;
+	max-width: 100%;
+
+	@include break-large {
+		max-width: 360px;
+	}
+
+	// TODO: Migrate to the checklist components. The checklist could receive a special className as prop that would be responsible for this.
```

Comment:
For now, I kept it as it is, but it would be nice to change it at some point to be a responsibility of the checklist.